src/main/kotlin/datamaintain/Script.kt

```diff
@@ -4,11 +4,13 @@ package datamaintain
 interface Script {
     val name: String
     val checksum: String
+    val tags: Set<Tag>
 }
 
 data class ScriptWithoutContent(
         override val name: String,
-        override val checksum: String
+        override val checksum: String,
+        override val tags: Set<Tag> = setOf()
```

Comment:
Tags are used to filter scripts that are ``ScriptWithContent`` and I don't see any user case that needs storing a script's tags in the database, which is what will happen if ``ScriptWithoutContent`` does contain tags. I suggest removing this field from ``ScriptWithoutContent`` and ``Script`` for tags to remain only in ``ScriptWithContent``.

Reply:
done